org.gottcode.FocusWriter.json

```diff
@@ -0,0 +1,35 @@
+{
+	"app-id": "org.gottcode.FocusWriter",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "focuswriter",
+	"rename-icon": "focuswriter",
+	"rename-desktop-file": "focuswriter.desktop",
+	"rename-appdata-file": "focuswriter.appdata.xml",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=home",
+		"--share=network"
+	],
+	"modules": [
+		{
+			"name": "focuswriter",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "git",
+					"url": "git://github.com/gottcode/focuswriter",
```

Comment:
It's not a rule – but I've had problems with git protocol on our build infra before. `https` might be safer.

Reply:
Done